projects/batfish/src/main/antlr4/org/batfish/grammar/cisco/Cisco_bgp_nxos.g4

```diff
@@ -621,7 +630,7 @@ rbnx_template_peer_policy
 rbnx_template_peer_session
 :
     TEMPLATE PEER_SESSION session = variable NEWLINE
-    rbnx_n_inner*
+    rbnx_n_common*
```

Comment:
fyi this is the main desired outcome -- now you cannot enter a-f inside a template peer session